src/Space.ts

```diff
@@ -54,7 +69,11 @@ class SpaceService extends Base {
   async getMany(queryString?: string): Promise<Space[]> {
     const { apiEndpointAddress, got } = this.lensPlatformClient;
     const url = `${apiEndpointAddress}/spaces${queryString ? `/?${queryString}` : ""}`;
-    const json = await got.get(url);
+
+    const json = await throwExpected(
+      () => got.get(url),
+      {}
```

Comment:
We could make this parameter optional, defaulting to `{}`.

In that case `throwExpected` seems unnecessary? (But maybe ok for consistency).

Reply:
Optional mapping is a great idea.
I've included `throwExpected` everywhere because it also handles unexpected exceptions by converting them to custom `LensSDKException` with debugging info.